### PR TITLE
Remove and Update topics.

### PR DESCRIPTION
**Pull request by MichelDiz**
- Remove "expand and _predicate_"
>We already have a "Expand" topic and "`_predicate_`" was removed from Dgraph core.
- Update topic according to the new Type System.
- Remove "`_predicate_`" example

### CODE REVIEW

File: content/moredata/2.md [hunk 1]
```diff
@@ -4,25 +4,31 @@ next = "/blocksvars/1"
...
 
-Dgraph queries and the visualization will help us understand the schema of the movies dataset.  
+Dgraph queries, Schema Types and the visualization will help us understand the schema of the movies dataset.  
 
 As you'll see in the following pages, we can view the data from a number of perspectives; considering for example directors, movies, genres or even locations as the primary starting point.
 
 Let's have a look from the perspective of directors.  Kathryn Bigelow directed the early 90's classic Point Break.  We'll begin with her and learn about the schema.
 
-Run the query.  `_predicate_` tells us that directors have a `name` and are connected to the movies they directed via `director.film`.
+Checking the Type Schema, it tells us that directors as a "Person Type" have a `name` and are connected to the movies they directed via `director.film`.
```
Comment (mangalaman93) [inline content/moredata/2.md]: Checking the Type Schema tells us

File: content/moredata/2.md [hunk 2]
```diff
@@ -4,25 +4,31 @@ next = "/blocksvars/1"
...
 
 Extend the query a level deeper by also including
 ```
-director.film (first: 1) { _predicate_ }
+director.film (first: 1) {
```
Comment (mangalaman93) [inline content/moredata/2.md:20]: We should query the type schema here.

File: content/moredata/2.txt [hunk 3]
```diff
@@ -1,6 +1,26 @@
...
+}
+
+type Performance {
+    performance.film: [Movie]
+    performance.character_note: string
+    performance.character: [Person]
+    performance.actor: [Person]
+    performance.special_performance_type: [Special_performance_type]
+    type: [Generic]
+}
```
Comment (mangalaman93) [inline content/moredata/2.txt]: Newline missing

File: content/moredata/2.md (hunk 2 above)
Comment (MichelDiz) [reply content/moredata/2.md:20]: But this isn't a query. Is an edge (a nested block). Do you think I should revamp this part as a query?

Comment (MichelDiz) [reply content/moredata/2.md:20]: I'll merge and solve this after.

**Pull request merged**